File: Arrays/C++/Linear Search.cpp
#include<iostream>
using namespace std;
int main()
{
    int arr[] = {1,2,3,4,5};
    int el = 3;
    int flag = -1;
    int n = sizeof(arr)/sizeof(arr[0]);
    for(int i=0;i<n;i++)
    {
        if (arr[i] == el)
        {
            flag = i;
            break;
        }
    }
    cout<<flag<<endl;
}
// Time Complexity -> O(n)
// Space Complexity -> O(1)

File: Arrays/Linear Search.py
"""
Docstring for Arrays.Linear Search

Recursive Linear Search Solution

def recurisive_linear_search(L,i,target):
    if L[i] ==len(L):
        return False
    if L[i] == target:
        return True 
    return recursive_linear_search(L,i+1,target)

Time Comeplexity : O(n)
Space Complexity : O(1)

"""

L= eval(input("Enter the Number : "))
target = int(input("Enter the target Element : "))
for i in range(len(L)):
    if L[i] == target:
        print(i)
        break 
else:   
    print(-1)